### ISSUE TITLE
[Bug]: Comment downloading break down

### ISSUE THREAD

**Issue by PhuriousGeorge** [tubearchivist/tubearchivist#383]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

Unraid

## Describe the bug
Comment downloading broke-down.  Looks like maybe a parsing issue?

### Relevant log output

```shell
[2022-12-07 07:08:45,085: WARNING/ForkPoolWorker-1] [youtube]     Downloading comment API JSON reply thread 3 (371/2051)
[2022-12-07 07:08:45,223: WARNING/ForkPoolWorker-1] ERROR: Incomplete data received
[2022-12-07 07:08:45,223: WARNING/ForkPoolWorker-1] j_OPGdL0wYo: failed to get info from youtube
[2022-12-07 07:08:45,228: ERROR/ForkPoolWorker-1] Task download_pending[98171cb5-e3e4-4bcf-95bf-858fd8bd4ab9] raised unexpected: AttributeError("'bool' object has no attribute 'get'")
Traceback (most recent call last):
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 90, in download_pending
    downloader.run_queue()
  File "/app/home/src/download/yt_dlp_handler.py", line 240, in run_queue
    DownloadPostProcess(self).run()
  File "/app/home/src/download/yt_dlp_handler.py", line 43, in run
    self.get_comments()
  File "/app/home/src/download/yt_dlp_handler.py", line 152, in get_comments
    comment.build_json(notify=(idx, total_videos))
  File "/app/home/src/index/comments.py", line 35, in build_json
    comments_raw, channel_id = self.get_yt_comments()
  File "/app/home/src/index/comments.py", line 99, in get_yt_comments
    comments_raw = info_json.get("comments")
AttributeError: 'bool' object has no attribute 'get'
```

**Comment by bbilly1** [comment]
Looks like yt-dlp failed first, unable to get the info-json: `ERROR: Incomplete data received`. Need to do some error handling so it can skip it in that case. 

I assume that video fixed itself on second try?

**Comment by PhuriousGeorge** [comment]
Due to no YTID in the logs, there's no way of telling which of the ~1500 videos that were downloaded and in queue for post-processing failed during comment downloading, nor a way to give it a second try (knowingly)?

Agreed though, a graceful failure and move to the next item in queue would be preferable ;)

**Comment by PhuriousGeorge** [comment]
Just ran into an error again.  This time it was a single-video queue and the issue appears to be more obvious from the log.  The video is a "members only" video and it appears maybe cookies are not used when grabbing comments?

I'm suspecting like you mentioned, the overall error shown in TA itself is generic and can happen for a number of reasons, but this one actually provided some context:

```[2022-12-09 00:38:21,024: WARNING/ForkPoolWorker-32] xYePANAObI0: get comments
[2022-12-09 00:38:21,157: WARNING/ForkPoolWorker-32] [youtube] xYePANAObI0: Downloading webpage
[2022-12-09 00:38:21,667: WARNING/ForkPoolWorker-32] [youtube] xYePANAObI0: Downloading android player API JSON
[2022-12-09 00:38:21,825: WARNING/ForkPoolWorker-32] ERROR: [youtube] xYePANAObI0: Join this channel to get access to members-only content like this video, and other exclusive perks.
[2022-12-09 00:38:21,825: WARNING/ForkPoolWorker-32] xYePANAObI0: failed to get info from youtube
[2022-12-09 00:38:21,829: ERROR/ForkPoolWorker-32] Task home.tasks.download_single[783b7b6d-6268-49db-bd1c-310c3706d669] raised unexpected: AttributeError("'bool' object has no attribute 'get'")
Traceback (most recent call last):
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 120, in download_single
    VideoDownloader().run_queue()
  File "/app/home/src/download/yt_dlp_handler.py", line 240, in run_queue
    DownloadPostProcess(self).run()
  File "/app/home/src/download/yt_dlp_handler.py", line 43, in run
    self.get_comments()
  File "/app/home/src/download/yt_dlp_handler.py", line 152, in get_comments
    comment.build_json(notify=(idx, total_videos))
  File "/app/home/src/index/comments.py", line 35, in build_json
    comments_raw, channel_id = self.get_yt_comments()
  File "/app/home/src/index/comments.py", line 99, in get_yt_comments
    comments_raw = info_json.get("comments")
AttributeError: 'bool' object has no attribute 'get'```

**Comment by bbilly1** [comment]
That should now be fixed in v0.3.1, please update. Thanks for reporting.
